src/Api/Struct/Certificate/Info.php

```diff
@@ -9,12 +9,28 @@ class Info extends AbstractStruct
 {
     public string $request;
     public string $privateKey;
+    public string $body;
 
-    public function __construct(\SimpleXMLElement $apiResponse)
+    public function __construct($input)
     {
-        $this->initScalarProperties($apiResponse, [
-            ['csr' => 'request'],
-            ['pvt' => 'privateKey'],
+        if (!is_array($input)) {
+            $this->initScalarProperties($input, [
+                ['csr' => 'request'],
+                ['pvt' => 'privateKey'],
+            ]);
+        } else {
+            foreach ($input as $name => $value) {
+                $this->$name = $value;
+            }
+        }
+    }
+
+    public function getMapping(): array
+    {
+        return array_filter([
+            'csr' => $this->request,
+            'pvt' => $this->privateKey,
+            'cert' => $this->body ?? null,
```

Comment:
change variable declaration `public ?string $body = null` and you don't have to check it